### PR TITLE
ButtonGlobalOverrides

### PR DESCRIPTION
**Pull request by GoodThomas**

### CODE REVIEW

File: src/config/theme.js [hunk 1]
```diff
@@ -297,4 +270,41 @@ export const theme = {
 
 const bodhiTheme = createMuiTheme(theme);
 
+bodhiTheme.overrides = {
+  ...bodhiTheme.overrides,
+  MuiButton: {
```
Comment (dwalintukan) [inline src/config/theme.js]: move this back into `theme.overrides`. do it inline.

File: src/config/theme.js [hunk 2]
```diff
@@ -297,4 +270,41 @@ export const theme = {
...
+    root: {
+      borderRadius: px(paddingLg),
+      textTransform: 'none',
+      [bodhiTheme.breakpoints.down('xs')]: {
+        padding: '7px 8px',
```
Comment (dwalintukan) [inline src/config/theme.js]: lets use `paddingXs` all around.

File: src/config/theme.js (hunk 1 above)
Comment (GoodThomas) [reply src/config/theme.js]: I want to use 'breakpoints' in 'overrides', but I didn't find a better way.

Comment (dwalintukan) [reply src/config/theme.js]: got it :+1:

**Pull request merged**